.github/workflows/build.yml

```diff
@@ -26,8 +26,8 @@ jobs:
       fail-fast: false
       matrix:
         go:
-          - 1.18
-          - 1.19.10
+          - 1.20.7
```

Comment:
Go now also provides .0 versions, so we can use 1.21.0 here.


Wondering if we should keep 1.18 in the list as an "old" version to test against (for projects that consume the code as a library).

Although perhaps less important for the main (unreleased / unstable) branch

Reply:
It's out of support, they shouldn't be using it.

Reply:
Updated to 1.21.0. Will leave 1.18 out for now.